app/modules/auth/auth_service.py

```
@@ -133,52 +136,32 @@ async def check_auth(
                 if "uid" in decoded_token and "user_id" not in decoded_token:
                     decoded_token["user_id"] = decoded_token["uid"]
-                logging.info(
-                    "DEBUG: Successfully verified token for user: %s",
-                    decoded_token.get("user_id", decoded_token.get("uid", "unknown")),
-                )
-                logging.info(
-                    "DEBUG: Token email: %s",
-                    decoded_token.get("email", "unknown"),
-                )
                 request.state.user = decoded_token
             except Exception as err:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)